src/rewards/mod.rs

```diff
@@ -285,3 +396,41 @@ impl<'a> RewardsMonitor<'a> {
         }
     }
 }
+
+/// Calculates the staking APY of an `AccountInfo` containing a `StakeState`.
+/// Returns the calculated APY while registering the delegated voter in `seen_voters`
+/// for later reference.
+fn calculate_staking_apy(
+    account_info: &Account,
+    seen_voters: &mut BTreeSet<Pubkey>,
+    epoch_duration: f64,
+    lamports: u64,
+    post_balance: u64,
+) -> anyhow::Result<Option<StakingApy>> {
+    let stake_state: StakeState = bincode::deserialize(&account_info.data)?;
+    if let Some(delegation) = stake_state.delegation() {
+        let percent = if !seen_voters.contains(&delegation.voter_pubkey) && lamports > 0 {
+            let prev_balance = post_balance - lamports;
+            let epoch_rate = lamports as f64 / prev_balance as f64;
+            let apr = epoch_rate / epoch_duration * 365.0;
+            let epochs_in_year = 365.0 / epoch_duration;
```

Comment:
Using 365.0 more than once qualifies it for a `const` as well IMO.

Reply:
Fixed in 78907158